tests/test_main.py

```diff
@@ -7,26 +9,75 @@
 
 
 def test_get_root():
+    """test the root test node"""
     response = client.get("/")
     assert response.status_code == 200
     assert "version" in response.json()
 
 
+# Endpoint #1
 @pytest.fixture
 def test_get_vote_store_id():
+    """Test retreiving and printing a voter store id guid"""
     response = client.get("/vote/")
     assert response.status_code == 200
     assert "VoteStoreID" in response.json()
     # retrieve VoteStoreID from response
     return response.json()["VoteStoreID"]
 
 
+# Endpoint #2
+@pytest.fixture
 def test_get_empty_ballot(test_get_vote_store_id):
-    # test with invalid VoteStoreID
+    """Test with invalid VoteStoreID"""
     response = client.get("/vote/00000X")
     assert response.status_code == 200
     assert "error" in response.json()
     # test with valid VoteStoreID
     response = client.get(f"/vote/{test_get_vote_store_id}")
     assert response.status_code == 200
-    assert "ballot" in response.json()
+    assert "blank-ballot" in response.json()
+    return response.json()["blank-ballot"]
```

Comment:
Can I duck on this one?  I am unfamiliar with the notions of fixture and tests - I was just tweaking what Neil started.  Apologies if I messed it up, but I would prefer to defer to whatever you guys want.  If you can 'make-it-so' so that I can understand the design target, I can then extend what is there.

Reply:
Yes, you can duck on it. In fact as I realized towards the end of doing it you almost certainly have to. I was most of the way through when I realized this needs to be run, and there's no way to test it here. It needs its own PR.

Reply:
Creating issue https://github.com/TrustTheVote-Project/VTP-web-api/issues/23 to track this